kafka/versions.py

```diff
@@ -18,7 +18,7 @@
         "jmx_exporter": "1.1.0",
     },
     {
-        "product": "3.8.1",
+        "product": "3.8.0",
```

Comment:
As discussed via Slack this should be reverted

Reply:
Related PR: https://github.com/stackabletech/kafka-operator/pull/834

Reply:
Thanks!